src/github.js

```diff
@@ -112,6 +112,20 @@ async function assign_reviewers(reviewers) {
   });
 }
 
+// https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28#list-team-members
+async function get_team_members(team) {
+  const context = get_context();
+  const octokit = get_octokit();
+
+  console.log(context);
+  const { data: response_body } = octokit.teams.listMembersInOrg({
+    org: context.repo.org,
+    team_slug: team,
+  });
+
+  return response_body.map((member) => member.login);
```

Comment:
TODO 1: I haven't tested if this works yet. There's no easy way to test it, so I may as well use this action in another repo to see if it really works...